dotnet-authserver/src/TeacherIdentity.AuthServer/Migrations/TeacherIdentityServerDbContextModelSnapshot.cs

```diff
@@ -235,7 +235,11 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("jsonb")
                         .HasColumnName("payload");
 
-                    b.HasKey("event_id")
+                    b.Property<bool>("Published")
+                        .HasColumnType("boolean")
+                        .HasColumnName("published");
+
+                    b.HasKey("EventId")
```

Comment:
These updates relate to an earlier migration, but NuGet wouldn't let me create a blank migration without including them.